web_console/creating-quick-start-tutorials.adoc

```diff
@@ -1,8 +1,17 @@
+ifndef::openshift-rosa,openshift-dedicated[]
 :_mod-docs-content-type: ASSEMBLY
 [id="creating-quick-start-tutorials"]
 = Creating quick start tutorials in the web console
 include::_attributes/common-attributes.adoc[]
 :context: creating-quick-start-tutorials
+endif::openshift-rosa,openshift-dedicated[]
```

Comment:
Is the only difference here in the title? Suggest conditionalizing just the title if so.

Reply:
Also, should the "Creating quick start tutorials in the web console" section be in the build output for openshift-enterprise? I don't see it. 

If at all possible, I would suggest not conditionalizing the topic title and preamble. It makes the content difficult to maintain.

Reply:
| Suggest conditionalizing just the title if so.

@aireilly I couldn't get it to work any other way than this.

Reply:
Here is the assembly in [OCP](https://66934--docspreview.netlify.app/openshift-enterprise/latest/web_console/creating-quick-start-tutorials).